src/lib/viewers/BaseViewer.js

```diff
@@ -915,7 +917,9 @@ class BaseViewer extends EventEmitter {
 
     disableAnnotationControls() {
         if (this.annotator && this.annotationControls && this.areNewAnnotationsEnabled()) {
-            this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
+            this.annotator.toggleAnnotationMode(
```

Comment:
Something that I noticed is that we have two toggleAnnotationMode method called being made in the 
`handleFullscreenEnter` -> `disableAnnotationControls` flow. My thinking is that either the first or second is redundant?

Reply:
Good point. I think the first is redundant.

Reply:
I'm not sure. It seems like these changes somewhat defeat the purpose of the original code, which was to forcibly disallow annotation creation. What effect does this have on creating annotations on rotated images, for example?

Reply:
My change was not in the right location, so I have moved it over to handleFullScreenExit instead.